app/cdap/components/GridTable/index.tsx

```diff
@@ -84,6 +82,8 @@ export default function GridTable() {
     },
   ]);
   const [snackbarState, setSnackbar] = useSnackbar();
+  const [recipeDetails, setRecipeDetails] = useState<IRecipeItem>();
```

Comment:
can this be recipe, setRecipe
and details are nothing but a recipe

pl update and let me know if u think otherwise with proper explanation

Reply:
Removed this variable from here as not needed but further wherever we will need it we will replace it with above mentioned name